### ISSUE TITLE
Thermostat 'set' topic not working correctly

### ISSUE THREAD

**Issue by Aragur** [mikejac/node-red-contrib-google-smarthome#67]
Hey,

if I set the Thermostat via the `set` topic, using a complete payload object, my `thermostatTemperatureAmbient` isn't set correctly in the Google-Home App. (10, the default value is shown there).

```json
{"thermostatTemperatureSetpoint":"20.5","thermostatTemperatureAmbient":"19.5","online":true}
```
If I set it via the `thermostatTemperatureAmbient` topic, it works correctly.

Nothing at the Google site has changed ([I think](https://developers.google.com/assistant/smarthome/guides/thermostat))

**Comment by Caprico85** [comment]
You are right. "set" was the only topic that did not work. Every other topic (or no topic) would have worked.

I released v0.0.47 which fixes that problem. "set" should now work.

**Comment by Aragur** [comment]
I'm getting the same Error.
```"Error: Type of thermostatTemperatureAmbient is not compatible; typeof = undefined"```

**Comment by Caprico85** [comment]
Thank you, merged in v0.0.48.
